### ISSUE TITLE
How to single long string ?

### ISSUE THREAD

**Issue by edmondo1984** [wemake-services/wemake-python-styleguide#2473]
### What's wrong

When you have a long string like the following:

`_my_message = 'very long string' `

and it exceeds the length of the line, one can break it into two strings like so

```
_my_message = ('very long'
      ' string')
```

but one would get an implicit concatenation error  WPS326 Found implicit string concatenation

and if you use a +, you get another violation such as W503 line break before binary operator or W504 line break after binary operator

### How it should be

There should be a way to break long strings into multi-line and concatenating without having to use noqa

### Flake8 version and plugins

```
❯ flake8 --bug-report
{
  "dependencies": [],
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.9.7",
    "system": "Darwin"
  },
  "plugins": [
    {
      "is_local": false,
      "plugin": "flake8-bandit",
      "version": "3.0.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-broken-line",
      "version": "0.4.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-bugbear",
      "version": "22.7.1"
    },
    {
      "is_local": false,
      "plugin": "flake8-comprehensions",
      "version": "3.10.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-darglint",
      "version": "1.8.1"
    },
    {
      "is_local": false,
      "plugin": "flake8-debugger",
      "version": "4.1.2"
    },
    {
      "is_local": false,
      "plugin": "flake8-docstrings",
      "version": "1.6.0, pydocstyle: 6.1.1"
    },
    {
      "is_local": false,
      "plugin": "flake8-eradicate",
      "version": "1.2.1"
    },
    {
      "is_local": false,
      "plugin": "flake8-string-format",
      "version": "0.3.0"
    },
    {
      "is_local": false,
      "plugin": "flake8_commas",
      "version": "2.1.0"
    },
    {
      "is_local": false,
      "plugin": "flake8_isort",
      "version": "4.2.0"
    },
    {
      "is_local": false,
      "plugin": "flake8_quotes",
      "version": "3.3.1"
    },
    {
      "is_local": false,
      "plugin": "mccabe",
      "version": "0.6.1"
    },
    {
      "is_local": false,
      "plugin": "naming",
      "version": "0.12.1"
    },
    {
      "is_local": false,
      "plugin": "pycodestyle",
      "version": "2.8.0"
    },
    {
      "is_local": false,
      "plugin": "pyflakes",
      "version": "2.4.0"
    },
    {
      "is_local": false,
      "plugin": "rst-docstrings",
      "version": "0.2.7"
    },
    {
      "is_local": false,
      "plugin": "wemake_python_styleguide",
      "version": "0.16.1"
    }
  ],
  "version": "4.0.1"
}
```

### pip information

irrelevant

### OS information

macos

**Comment by sobolevn** [comment]
It is:

```python
_my_message = (
    'very long' +
    ' string'
)
```

🙂 

Please, feel free to reopen if you have other questions!

**Comment by edmondo1984** [comment]
That won't work:

./metal_wrapper/remote_path.py:27:44: W504 line break after binary operator

**Comment by sobolevn** [comment]
It works with `wemake-python-styleguide 0.16.1` and ignoring `W504`

**Comment by edmondo1984** [comment]
Why is W504 ignored?

**Comment by skarzi** [comment]
Generally, you need to ignore one of the following flake8 issues related to new like break and operators: W503 or W504
